examples/Videos/Sample/ViewController.h

```diff
@@ -14,8 +14,7 @@
 //  ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 //  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
-#import <AsyncDisplayKit/AsyncDisplayKit.h>
-#import <AsyncDisplayKit/ASVideoNode.h>
+#import <UIKit/UIKit.h>
 
 @interface ViewController : UIViewController
```

Comment:
Interesting.  Technically this should be an ASViewController, but of course as-is it makes for a useful test case for this...although a bit of a "secret" purpose / functionality!